frontend/src/ui-kit/components/Layout/MobileNav.tsx

```
@@ -0,0 +1,264 @@
+            isExpanded ? 'rotate-0' : '-rotate-90'
+          }`}
+          aria-hidden="true"
+        />
+      </button>
+
+      <div
+        id={contentId}
+        className={`mt-1 space-y-0.5 overflow-hidden transition-all duration-200 ${
+          isExpanded ? 'max-h-screen opacity-100' : 'max-h-0 opacity-0'
+        }`}
+        role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)